pkg/algorithm/hived_algorithm.go

```diff
@@ -913,10 +914,15 @@ func (h *HivedAlgorithm) scheduleGuaranteedAffinityGroup(
 	common.SortInt32(gpuNums)
 	lazyPreemptedGroups := h.tryLazyPreempt(virtualPlacement, gpuNums, sr.affinityGroupName)
 	preassignedCells, nonPreassignedCells := virtualPlacement.toBindingPaths(gpuNums, bindings)
+	freeCellNum := map[CellLevel]int32{}
```

Comment:
freeCellNumCopy?

Reply:
and maybe comment why we need a copy (we will make temporary changes to the data structure in the below function call)

Reply:
updated